pkg/exporter/exporter.go

```diff
@@ -58,9 +58,12 @@ func (e *Exporter) Send(event *tetragon.GetEventsResponse) error {
 		e.rateLimiter.Drop()
 		return nil
 	}
+
 	if err := e.encoder.Encode(event); err != nil {
 		logger.GetLogger().WithError(err).Warning("Failed to JSON encode")
 	}
+	eventsExportedTotal.Inc()
```

Comment:
AFAICS Write is called for event, so this could be called from the byteCounterWriter.Write?

Reply:
Depending on the underlying encoder and writer and the event being encoded, it may take multiple writes for a single event, so that would be inaccurate.